answer/templates.html

```diff
@@ -17,6 +17,10 @@
 </script>
 <!-- runtimeFilters -->
 <script type="text/template" id="runtimeFilters">
-     runtimeFilters: [],
+     runtimeFilters: [{
+          columnName: 'color',
```

Comment:
Let's add it as columnName1 instead of color and value1 instead of red, to give it as an example only.

Update in other usages as well.